src/style.css

```diff
@@ -0,0 +1,24 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
```

Comment:
Not last of type. Remove last margin 
![image](https://github.com/mate-academy/layout_stars/assets/70009205/fc799e1c-e55b-472c-aaae-2012c0716736)